src/test/java/io/jenkins/plugins/coverage/model/CoveragePluginITest.java

```diff
@@ -0,0 +1,264 @@
+package io.jenkins.plugins.coverage.model;
+
+import java.io.IOException;
+
+import org.junit.Rule;
+import org.junit.Test;
+import org.jvnet.hudson.test.JenkinsRule;
+
+import org.jenkinsci.plugins.workflow.cps.CpsFlowDefinition;
+import org.jenkinsci.plugins.workflow.job.WorkflowJob;
+
+import hudson.model.HealthReport;
+import hudson.model.HealthReportingAction;
+import hudson.model.Result;
+import hudson.model.Run;
+import static io.jenkins.plugins.coverage.model.Assertions.*;
+
+import io.jenkins.plugins.checks.api.ChecksAction;
+import io.jenkins.plugins.util.IntegrationTestWithJenkinsPerSuite;
+
+
+
+
+/**
+ * Integration test for the coverage API plugin.
+ *
+ * @author Thomas Willeit
+ *
+ */
+public class CoveragePluginITest extends IntegrationTestWithJenkinsPerSuite {
+
+    private static final String JACOCO_FILE_NAME = "jacoco-analysis-model.xml";
+    private static final String JACOCO_LESS_FILE_NAME = "jacoco-codingstyle.xml";
+    private static final String COBERTURA_FILE_NAME = "cobertura.xml";
+    private static final String COBERTURA_FILE_NAME_2 = "cobertura2.xml";
+
+
+    @Rule
+    public JenkinsRule j = new JenkinsRule();
```

Comment:
This creates another Jenkins instance *per test*. Remove it, it is already in the base class.